include/seqan3/search/configuration/detail.hpp

```diff
@@ -39,14 +39,14 @@ namespace seqan3::detail
  */
 enum struct search_config_id : uint8_t
 {
-    //!\brief Identifier for max_errors configuration.
-    max_error,
-    max_error_rate,
-    output,
-    mode,
+    max_error, //!< Identifier for max_errors configuration.
+    max_error_rate, //!< Identifier for max_errorrate configuration.
```

Comment:
`max_errorrate` ? is that a single word?

Reply:
:see_no_evil: of course not